examples/vue-router/app.js

```diff
@@ -15,30 +22,40 @@ const ChildComponent = {
 <h3>You're looking at the <strong>{{ page }}</strong> page</h3>
 <p>Has metaInfo been updated due to navigation? {{ metaUpdated }}</p>
 </div>`,
-  metaInfo () {
-    return {
-      title: `${this.page} - ${this.date && this.date.toTimeString()}`,
-      bodyAttrs: {
-        class: 'child-component'
-      },
-      afterNavigation () {
-        metaUpdated = 'yes'
-      }
-    }
+  metaInfo: {
+    title() {
+       return `${this.page} - ${this.date && this.date.toTimeString()}`
+    },
+    /*bodyAttrs: {
+      class: 'child-component'
+    },*/
+    meta: [
+      function () { return { vmid: 'descr', name: 'description', content: `description at ${this && this.date2 && this.date2.toTimeString()}` } },
```

Comment:
Why `this && ...` @pimlie ?

I believe  `this.date2 && this.date2.toTimeString()` should be enough right?

Reply:
That was a left over for easily testing all the different metainfo use-cases we will have now. Was switching a lot from objects to functions during testing and this way I didnt have to care about this not existing when metaInfo was fully a plain object :)